resources/providers/awslib/ec2/ebs_encryption.go

```diff
@@ -0,0 +1,41 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package ec2
+
+import (
+	"fmt"
+
+	"github.com/elastic/cloudbeat/resources/fetching"
+)
+
+type EBSEncryption struct {
+	Enabled bool
```

Comment:
```suggestion
	Enabled bool `json: enabled`
```

Reply:
not sure why

Reply:
Not a must but it will be cleaner to address this field in rego like:
`input.resource.enabled`
rather than
`input.resource.Enabled`